src/main/java/ch/epfl/biop/operetta/OperettaManager.java

```diff
@@ -729,6 +757,34 @@ public ImagePlus getFieldImage(WellSample field, Roi subregion) {
         return result;
     }
 
+    public List<Channel> getChannels(int fieldIndex, List<Integer> channels){
+        List<Channel> chList = new ArrayList<>();
+        List<Channel> originalChannels = ((OMEXMLMetadataRoot) metadata.getRoot())
+                .getImage(fieldIndex)
+                .getPixels()
```

Comment:
Is getPixels reading reading the pixels ? Is it a performance issue in that case ? Is getChannels called often ?

Reply:
I think it reads the ome.xml.Pixel object. getPixels() return the value of 'this.pixels', which has been read previously, when the images has been open. So, it shouldn't be a performance issue.
getChannels() is called for each fused image

Reply:
alright